File: graphs/dfs/py/run_dfs.py
import dfs

def main():
    g = dfs.Graph(5)
    g.add_edge(0, 1)
    g.add_edge(1, 3)
    g.add_edge(1, 2)
    g.add_edge(3, 4)
    g.print()
    d = dfs.DFS(g)
    d.dfs(0)

main()

File: graphs/dfs/cpp/run_dfs.cpp
#include "dfs.h"

#include <iostream>

int main()
{
    auto g = Graph(5);
    std::cout << "hello" << std::endl;
    g.add_edge(0, 1);
    g.add_edge(1, 3);
    g.add_edge(1, 2);
    g.add_edge(3, 4);
    g.print();
    auto dfs = DFS(g);
    dfs.dfs(0);
}